File: DailyFlash Solutions/DailyFlash Solutions/03-DailyFlash_Solutions/16_Jan_Solutions_Three/Java/prog5.java
import java.io.*;
class Demo{
	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		System.out.println("Input : ");
		int p = Integer.parseInt(br.readLine());
		int c = Integer.parseInt(br.readLine());
		int m = Integer.parseInt(br.readLine());
		int b = Integer.parseInt(br.readLine());
		int comp = Integer.parseInt(br.readLine());
		float per = (p+c+m+b+comp)/5;
		if(p>=0 && c>=0 && m>=0 && b>=0 && comp>=0 && p<=100 && c<=100 && m<=100 && b<=100 && comp<=100){
			if(per>=90){
				System.out.println("Grade A");
			}else if(per>=80 && per<90){
				System.out.println("Grade B");
			}else if(per>=70 && per<80){
				System.out.println("Grade C");
			}else if(per>=60 && per<70){
				System.out.println("Grade D");
			}else if(per>=40 && per<60){
				System.out.println("Grade E");
			}else if(per<40){
				System.out.println("Grade F");
			}
		} else {
			System.out.println("Enter valid marks");
		}
	}
}

File: DailyFlash Solutions/DailyFlash Solutions/03-DailyFlash_Solutions/16_Jan_Solutions_Three/Python/prog5.py
p, c, m, b, comp = [int(i) for i in input("Marks : ").split()]
per = (p+c+m+b+comp)/5
if p>=0 and c>=0 and m>=0 and b>=0 and comp>=0 and p<=100 and c<=100 and m<=100 and b<=100 and comp<=100:
    if per>=90:
        print("Grade A")
    elif per>=80 and per<90:
        print("Grade B")
    elif per>=70 and per<80:
        print("Grade C")
    elif per>=60 and per<70:
        print("Grade D")
    elif per>=40 and per<=60:
        print("Grade E")
    elif per<40:
        print("Grade F")
else:
    print("Enter valid marks")
